R/columns.R

```diff
@@ -93,8 +93,8 @@ get_default_id_label_map <- function(data, id_column, label_column = NULL,
   if (is.null(label_column)) {
     label_column <- id_column
   }
-  values <- unique(data[, c(id_column, label_column)])
-  if (length(unique(data[, id_column])) != nrow(values)) {
+  values <- dplyr::distinct(data, !!rlang::sym(id_column), !!rlang::sym(label_column))
+  if (nrow(dplyr::distinct(data, !!rlang::sym(id_column))) != nrow(values)) {
```

Comment:
Thanks, can you add `dplyr` and `rlang` into the `DESCRIPTION` please

Reply:
done